apps/publish/content/common.py

```diff
@@ -967,7 +967,8 @@ def _inherit_publish_schedule(self, original, updates, associated_item):
         if self.publish_type == "publish" and (updates.get(PUBLISH_SCHEDULE) or original.get(PUBLISH_SCHEDULE)):
             schedule_settings = updates.get(SCHEDULE_SETTINGS, original.get(SCHEDULE_SETTINGS, {}))
             publish_schedule = updates.get(PUBLISH_SCHEDULE, original.get(PUBLISH_SCHEDULE))
-            if publish_schedule and not associated_item.get(PUBLISH_SCHEDULE):
+            if publish_schedule:
```

Comment:
I wonder why this would fix it, seems like on deschedule it unsets the schedule on associations as well, doesn't it?